rpcclient/make_offer.go

```diff
@@ -13,14 +13,14 @@ func (c *Client) MakeOffer(
 	ethAsset types.EthAsset,
 	relayerEndpoint string,
 	relayerCommission float64,
-) (string, error) {
+) (*rpctypes.MakeOfferResponse, error) {
```

Comment:
Same on this one. We are returning the response now, because it includes both the Peer ID and the Offer ID. We want to return the Peer ID, as it is not specified in the request, so there is a good chance Bob doesn't have it. If Bob wants to pass the swap information on to Alice, he needs to pass both pieces of information for her to accept the offer.